docs/preview/index.md

```diff
@@ -25,7 +25,8 @@ For more granular packages we recommend reading the documentation.
 * Automate Azure Logic Apps tasks ([powershell](features/powershell/azure-logic-apps))
 * Automate Azure Resource Manager (ARM) tasks ([powershell](features/powershell/arm))
 * Automate Azure Security tasks ([powershell](features/powershell/azure-security))
-* Automate Azure Storage tasks ([powershell](features/powershell/azure-storage))
+* Automate Azure Storage (File Share) tasks ([powershell](features/powershell/azure-storage-fileshare))
+* Automate Azure Storage (Table) tasks ([powershell](features/powershell/azure-storage-table))
```

Comment:
Over here we should then link to the metapackage-documentation page (Arcus.Scripting.Storage), which links on towards the Table/FileShare-specific documentation pages.

Reply:
Yes, did that. Maybe here also quick-link to the other storages to find directly the one you seek?

Reply:
Yes, good idea!